inst/templates/ad_adbcva.R

```diff
@@ -280,19 +280,25 @@ adbcva_adsl <- adbcva_aseq %>%
     by_vars = get_admiral_option("subject_keys")
   )
 
-adbcva_crtflag <- adbcva_adsl %>%
-  # Add criterion flags for BCVA endpoints
-  restrict_derivation(
-    derivation = derive_var_bcvacritxfl,
-    args = params(
-      crit_var = exprs(CHG),
-      bcva_ranges = list(c(0, 5), c(-5, -1), c(10, 15)),
-      bcva_uplims = list(-20, 5, 10),
-      bcva_lowlims = list(-15, 15),
-      additional_text = ""
-    ),
-    filter = PARAMCD %in% c("SBCVA", "FBCVA")
-  ) %>%
+
+# Add criterion flags for BCVA endpoints
+adbcva_crtflag <- call_derivation(
```

Comment:
Please could you move this section before the ASEQ derivation? Even though it makes no difference to the final output I think it makes logical sense to derive ASEQ as the final variable.